index.html

```diff
@@ -5,6 +5,7 @@
     <meta charset="UTF-8">
     <title>Karim Portfolio</title>
     <link href="style.css" rel="stylesheet" type="text/css">
+	<link href="normalize.css" rel="stylesheet" type="text/css">
```

Comment:
unnecessary space.
I think the proper order for link tag attributes is `rel`, `type`, `href`. I might be wrong though...

Reply:
not sure why the space is there. Looks ok on my IDLE